TemporaryProjects.14.0/TemporaryProjects.14.0.csproj

```diff
@@ -105,12 +105,12 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="..\TemporaryProjects.Vsix\source.extension.vsixmanifest">
```

Comment:
Should this be `None`? Maybe the link would work then.

Reply:
The issue turned out to be that the `CreateVsixContainer` and `DeployExtension` properties hadn't been configured for the `Release` configuration. CI was attempting a `Release` build which was failing because the `TemporaryProjects.14.0` shouldn't have been creating a VSIX (which requires a `source.extension.vsixmanifest`).

Reply:
I see, thanks